src/server_manager/ui_components/outline-server-settings.html

```diff
@@ -203,13 +284,18 @@ <h3>[[localize('settings-metrics-header')]]</h3>
         serverManagementApiUrl: {type: String, value: null},
         serverPortForNewAccessKeys: {type: Number, value: null},
         isAccessKeyPortEditable: {type: Boolean, value: false},
+        isAccessKeyDataLimitEnabled: {type: Boolean, notify: true},
+        accessKeyDataLimit: {type: Object, value: null}, // type: app.DisplayDataAmount
+        canSetAccessKeyDataLimit: {type: Boolean, value: false},
+        dataLimitsAvailabilityDate: {type: String},
         serverCreationDate: {type: String, value: null},
         serverLocation: {type: String, value: null},
         serverMonthlyCost: {type: String, value: null},
         serverMonthlyTransferLimit: {type: String, value: null},
         localize: {type: Function, readonly: true},
         enableKeysPortButtons: {type: Boolean, value: false},
-        showKeysPortButtons: {type: Boolean, value: false}
+        showKeysPortButtons: {type: Boolean, value: false},
+        shouldShowExperiments: {type: Boolean, computed: '_computeShouldShowExperiments(canSetAccessKeyDataLimit)'},
```

Comment:
Can we just default this to false and set it in showServer where we set all the other properties anyways?

Reply:
We currently don't set any `outline-server-settings` properties in `showServer`. We use the bound properties from `outline-server-view`. We spoke offline about removing the duplication in a separate refactor.

I think computing the property from existing values removes the burden from setting an  additional property in `showServer`.